scripts/_vault_commands.py

```
@@ -0,0 +1,455 @@
+    "_generate_secret_id",
+    "_write_approle_policy",
+    "build_vault_env",
+    "collect_droplet_ips",
+    "ensure_approle",
+    "ensure_kv_engine",
+    "fetch_vault_status",
+    "initialise_vault",
+    "run_command",
+    "unseal_vault",
+    "verify_vault_service",
+]
```

Ruff (0.14.0)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)